UQL/Interpreter.php

```diff
@@ -280,10 +282,15 @@ private function parseValue($value)
      */
     private function callFunction(ASTFunctionCall $functionCall)
     {
+        $functionName = $functionCall->getFunctionName();
+        $arguments = $functionCall->getArguments();
+
+        $arguments[] = new Context($this->dataSource->getEntityClass());
```

Comment:
`new` in the middle of a class that is not a factory raises alarm bells form me. any nicer way?